src/services/CalculatorProcess.js

```diff
@@ -21,11 +23,11 @@ function CalculatorProcess(content, visor) {
 		memory = memory.slice(0, -1);
 
 		if (memory.includes("%")) {
```

Comment:
why it's inside of `=` condition?

Reply:
because this is equal to every simple math expression, i mean it's the same thing as 1+1;1-1;1/1, but the **%** operation it different, with him i want to return the percentage value and the **%** by default don't do this it returns the rest of division, that's it has a if for this.